data/human/human missions.txt

```diff
@@ -9341,3 +9342,799 @@ mission "Timothy Radrickson 7: Grave"
+			`	He nods to himself and becomes more confident. "Long ago we would find the Scales of Ran on the beaches nearest her God-Stone. The stories say that they made a bubble of breathable air around those who carried them, and allowed us to explore the secrets of the depths."`
+			choice
+				`	"What happened to them?"`
+				`	"Where could I find them?"`
+
+			`	Aldfrith shakes his head slowly. "In the time of my grandfather the men from the Deep came and took them off world. They said they were too valuable to belong in the hands of savages, and since then we've never found another Scale of Ran. I think she was displeased we allowed them to be taken."`
+			`	The old man pauses for a moment and then says, "Although... I do remember when I was a child the folk from the Deep coming and visiting us again. They had questions about... our baubles. I think they said they did not work? They thought they were just useless charms. Perhaps the Scales stopped working?"`
+			`	He shrugs. "I am sorry, <first>, that I cannot tell you more."`
+			`	The Ondurdis puts a hand on his back. "I value your stories." She turns her attention back to you, "Maybe all God-Stones once produced wondrous stones of power."`
+			`	She points at the Blood of Surtr. "Maybe that is where yours came from. But more than guesses, we cannot provide."`
+			choice
+				`	(Take your leave.)`
+
+			`	You thank the Ondurdis and Aldfrith for their gracious help and interesting stories. The Ondurdis hastily prepares a small meal, believing it bad manners to let a guest leave unfed, and wishes you well in your future travels among the lofty stars.`
+				decline
+
+
+mission "Stone to Hope: Thanks"
+	source "Norn"
+	to offer
+		has "Stone to Ice Queen: offered"
+	on offer
+		conversation
+			`In the quiet, creaking spaceport, it's easy enough to spot Helm doing some much needed maintenance on the ships that have landed here for the night. The last time you met him, he asked you to bring a stone to Hope. Do you want to check in with him?`
+			choice
+				`	(Go over to Helm and tell about the stone.)`
+					goto tell
+				`	(Leave Helm alone.)`
+					decline
+			
+			label tell
+			`	As you say your hello you accidentally startle Helm, who's in the middle of recalibrating a repulsor.`
+			`	"Woah! <first>! No need to sneak up on me!" He stands up to greet you. "What's new?"`
+			choice
+				`	"I found the Family-Cairn and left the stone there."`
+					goto where
+					to display
+						has "placed tosti's stone"
+				`	"I chucked the stone out of a hatch, but it's at least on Hope now."`
+					goto threw
+					to display
+						has "chucked tosti's stone"
+				`	"I threw the stone off a cliff, but it's at least on Hope now."`
+					goto threw
+					to display
+						has "threw tosti's stone"
+				`	"Where's Tosti?"`
+			label where
+			`	"Ah, Tosti's out on the seas. The lad got himself a job working on a fishing trawler. It'd likely be a few years before he'll call to port here again."`
```

Comment:
This first sentence feels a bit incongruous with the first dialog choice.

Reply:
I have made some edits to address this!